cfgov/ask_cfpb/scripts/migrate_spanish_short_answers.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import unicode_literals
+
+import os
+
+from django.contrib.auth.models import User
+
+from bs4 import BeautifulSoup as bs
+
+from ask_cfpb.models.pages import AnswerPage
+
+
+tags = ['em', 'i', 'strong', 'b']
+
+false_positives = [1637]
+
+
+def run():
+    answers = [
```

Comment:
I think that doing `AnswerPage.objects.filter(language='es', redirect_to=None, live=True)` here will accomplish the same thing since anything that is not a pure draft will be live.

Reply:
Updated in f6d56132469f010e6104ffa38f718c63f00d4bc1.